### DISCUSSION TITLE
Premisions

### DISCUSSION THREAD

**Discussion by Taconamedyoda** [GeorgH93/Minepacks#295] [Q&A]
I go in the game at first in op test out the command /backpack works wonderfully then I put myself in survival yes the accepted game mode is survival and only survival. The /backpack nor the backpack shortcut works it says I don't have the needed permissions and I don't know where to fix it. HELP please.

**Comment by GeorgH93** [comment]
By default the permission to use the plugin is only given to op users. If you would like normal players to be able to use the plugin, you will have to give them the `backpack.use` permission (or better give them the `backpack.user` permission to enable all none administrative features for them).
You probably also should give them a size permission like `backpack.size.6` for a 6 row backpack.

If you don't have a permissions plugin, please check [this faq entry](https://github.com/GeorgH93/Minepacks/wiki/FAQ#how-can-i-use-the-plugin-without-a-permission-plugin).
